src/index.js

```diff
@@ -1,10 +1,34 @@
 // import _ from 'lodash';
```

Comment:
**You did a great job your app is fetching API as expected and submit button is setled very well.**

However, there are some areas that you still need to work on:
- [ ] For starting please remove all unused commneted codes form your file!